JamfSync/Model/Jcds2Dp.swift

```diff
@@ -13,12 +13,19 @@ class Jcds2Dp: DistributionPoint {
     var urlSession: URLSession?
     var downloadTask: URLSessionDownloadTask?
     var dispatchGroup: DispatchGroup?
+    
+    // for aws uri encoding - BIG-RAT
+    var myCharacterSet = CharacterSet()
```

Comment:
Best to pick a name that describes the character set instead of "my". The rule of thumb is to make variable names descriptive enough that comments are unnecessary. So maybe something like "awsUriEncodingCharacterSet" and then remove the comment.